Content.Client/Audio/Jukebox/JukeboxMenu.xaml.cs

```diff
@@ -86,33 +71,43 @@ private void PlaybackSliderKeyUp(Slider args)
     /// </summary>
     public void Populate(IEnumerable<JukeboxPrototype> jukeboxProtos)
     {
-        MusicList.Clear();
-
         foreach (var entry in jukeboxProtos)
```

Comment:
This looks like it's just going to leak controls on prototype reloading?

Reply:
`MusicList.RemoveAllChildren()` is now called.